pkg/util/computeskus/computeskus_test.go

```diff
@@ -255,3 +255,42 @@ func TestIsRestricted(t *testing.T) {
 		})
 	}
 }
+
+func TestSupportedOSDisk(t *testing.T) {
+	for _, tt := range []struct {
+		name                string
+		vmSku               string
+		supportsPremiumDisk string
+		wantOSDisk          string
+	}{
+		{
+			"Premium disk supported on VMSize",
+			"premium_disk_supported",
+			"True",
+			premiumDisk,
+		},
+		{
+			"Premium disk not supported on VMSize",
+			"premium_disk_not_supported",
+			"False",
+			standardDisk,
+		},
```

Comment:
one last nit, here we should write the names of the fields.
[here](https://github.com/uber-go/guide/blob/master/style.md#initializing-structs) is the bit from the uber style guide. While we don't follow it 100% in the code, it is a nice resource to look at from time to time